pandas/io/formats/csvs.py

```diff
@@ -324,8 +351,13 @@ def _save(self):
 
             self._save_chunk(start_i, end_i)
 
-    def _save_chunk(self, start_i: int, end_i: int):
-
+    def _save_chunk(self, start_i: int, end_i: int) -> None:
+        """
+        Parameters
```

Comment:
Same here; I think shouldn't need to add docstrings if they only repeat the annotations